### PR TITLE
fix(sdk): add `fs.writeFile()` missing options

### LINKED ISSUE #4756: missing encoding options for `fs.writeFile`
### I tried this:

reading and writing a file with a specific encoding

```js
bring fs;

fs.readFile("/tmp/file", encoding: "binary");
fs.writeFile("/tmp/file", "", ); // can't pass file encoding
```

### This happened:

missing file encoding option

### I expected this:

have file encoding option

### Is there a workaround?

_No response_

### Component

SDK

### Wing Version

_No response_

### Node.js Version

_No response_

### Platform(s)

_No response_

### Anything else?

_No response_

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

### PR DESCRIPTION
**Pull request by garysassano**
Closes #4756

## Checklist

- [x] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [x] Description explains motivation and solution
- [x] Tests added (always)
- [ ] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

File: libs/wingsdk/src/fs/fs.ts [hunk 1]
```diff
@@ -17,7 +17,23 @@ export interface ReadFileOptions {
   readonly encoding?: BufferEncoding;
   /**
    * The `flag` can be set to specify the attributes.
-   * @default 'r'.
+   * @default `r``.
```
Comment (staycoolcall911) [inline libs/wingsdk/src/fs/fs.ts:20]: Let's keep consistency where we can - seems like we usually use double quotes for defining `@default`:
```suggestion
   * @default "r".
```

File: libs/wingsdk/src/fs/fs.ts [hunk 2]
```diff
@@ -17,7 +17,23 @@ export interface ReadFileOptions {
...
+ */
+export interface WriteFileOptions {
+  /**
+   * The character encoding utilized for file writing.
+   * @default "utf-8"
+   */
+  readonly encoding?: BufferEncoding;
+  /**
+   * The `flag` can be set to specify the attributes.
+   * @default 'w'.
```
Comment (staycoolcall911) [inline libs/wingsdk/src/fs/fs.ts:36]:
```suggestion
   * @default "w".
```

**Review by staycoolcall911 — APPROVED** [review]

**Review by staycoolcall911 — DISMISSED** [review]
Please add an sdk spec test [here](https://github.com/winglang/wing/blob/main/examples/tests/sdk_tests/fs/basic.test.w)

File: libs/wingsdk/src/fs/fs.ts (hunk 1 above)
Comment (garysassano) [reply libs/wingsdk/src/fs/fs.ts]: Thanks for reporting this, it was actually a typo when I copy-pasted the existing code.

**Review by tsuf239 — APPROVED** [review]

**Pull request merged**